app/models/provider.rb

```diff
@@ -310,6 +310,14 @@ def can_only_sponsor_student_visa?
     can_sponsor_student_visa && !can_sponsor_skilled_worker_visa
   end
 
+  def can_only_sponsor_skilled_worker_visa?
+    !can_sponsor_student_visa && can_sponsor_skilled_worker_visa
+  end
+
+  def cannot_sponsor_visas?
+    can_sponsor_student_visa == false && can_sponsor_skilled_worker_visa == false
+  end
+
```

Comment:
Are these methods used anywhere? 🤔

Reply:
Have spotted where `can_only_sponsor_skilled_worker_visa?` is used (ProviderHelper#visa_sponsorship_short_status), but I don't think `cannot_sponser_visas?` is used by any migrated code.

Reply:
i don't think so - will remove that one